app/views/companies/index.html.erb

```diff
@@ -1,11 +1,25 @@
-<h3>All Companies</h3>
+<div class="index">
+  <div class="title">
+    <h1>All Companies</h1>
+    <%= button_to 'New Company', new_company_path, method: "get", class: 'btn btn-large btn-info' %>
+  </div>
 
-<ul>
-  <% @companies.each do |company| %>
-  <li class=company_<%= company.id %>>
-    <%= link_to company.name, company_path(company) %>: 
-    <%= link_to "Edit", edit_company_path(company) %>
-    <%= link_to "Delete", company_path(company), method: "delete" %>
-  </li>
-  <% end %>
-</ul>
+  <div class="company-container container-fluid">
+    <% @companies.each do |company| %>
+    <div class="panel panel-default" class=company_<%= company.id %>>
+      <div class="panel-heading">
+        <h3 class="panel-title">
+          <%= link_to company.name, company_path(company) %>
+          <span class="badge"><%= company.jobs.count %></span>
+        </h3>
+      </div>
+      <div class="panel-body">
+        <%= button_to 'Contacts', company_path(company), method: "get", class: 'btn btn-large btn-success' %>
```

Comment:
this is cool! another way to handle buttons is to use a link_to (since it is a get) and style it as a button. This is more common